src/services/broadcast-service.js

```diff
@@ -148,8 +171,9 @@ class BroadcastService {
       reply.rawMessage = this.user.message
       reply.message = messageToSend
     } else if (target === 'NETWORK') {
-      if (this.user.voiceMemo !== undefined && this.user.voiceMemo !== '') {
-        uMessage = APIService.sendNetworkVoiceMemo(
+      if (this.user.voiceMemo) {
```

Comment:
Same thing here I think for whatever reason it doesn't work without checking undefined and empty

Reply:
new message service